### PR TITLE
GEOPY-1477: Remove factor of half on cpu count for n_cpu=disabled

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1477 - Remove factor of half on cpu count for n_cpu=disabled**

### CODE REVIEW

File: simpeg_drivers/components/data.py [hunk 1]
```diff
@@ -490,9 +494,15 @@ def observed_data_types(self):
 
     @staticmethod
     def check_tensor(channels):
-        tensor_components = ["xx", "xy", "xz", "yx", "zx", "yy", "zz", "zy", "yz"]
-        has_tensor = lambda c: any(k in c for k in tensor_components)
-        return any(has_tensor(c) for c in channels)
+        tensor_components = "|".join(
```
Comment (benk-mira) [inline simpeg_drivers/components/data.py:497]: What's the "|".join doing here?

File: simpeg_drivers/electricals/direct_current/two_dimensions/__init__.py [hunk 2]
```diff
@@ -14,6 +14,3 @@
 #  not be provided or otherwise made available to any other person.
 #
 # ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
-
-
-from .params import DirectCurrent2DParams
```
Comment (benk-mira) [inline simpeg_drivers/electricals/direct_current/two_dimensions/__init__.py:19]: restore

**Review by benk-mira — CHANGES_REQUESTED** [review]
couple things to check

File: simpeg_drivers/components/data.py (hunk 1 above)
Comment (domfournier) [reply simpeg_drivers/components/data.py:497]: That's a regular expression syntax

File: simpeg_drivers/electricals/direct_current/two_dimensions/__init__.py (hunk 2 above)
Comment (domfournier) [reply simpeg_drivers/electricals/direct_current/two_dimensions/__init__.py:19]: Why?

Comment (domfournier) [reply simpeg_drivers/electricals/direct_current/two_dimensions/__init__.py:19]: Ok so changing them all is bigger undertaking. To be dealt with on GEOPY-1495

**Review by benk-mira — DISMISSED** [review]

**Review by benk-mira — APPROVED** [review]

**Pull request merged**